gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -161,17 +168,18 @@ def __init__(
       compression_type=CompressionTypes.AUTO,
       validate=True,
       **kwargs):
+    # type: (str, str, bool, **str) -> None
```

Comment:
nit: Consider moving type hints to the line where the argument is declared for multi-line argument lists, like `read_records` function you have added above.

Reply:
Sometimes I couldn't decide whether to go for one line or multi-line type hints. Usually when the entire annotation cannot fit into one line, I will go for multi-line arguments. Do you have any kind of standard (for example, like when there are more than 3 arguments) for the multiple lines?

Reply:
I'm not aware of any 'standards' either, but I think in general it's just a judgement call about how easy it is to map args to their values. I think Bashir's suggestion was to always do inline when we have multi-line args, but I think that's an overkill in some situations like here where it's relatively easy to map types to args. There is another example where I suggested moving to inline though as there are many args and it's more difficult.

Reply:
Thanks for clarifying. I fixed that and some other similar methods.